### PR TITLE
Scorecardtest

### PR DESCRIPTION
**Pull request by rocrisp**
I fixed an error that I should have trapped. Basically Scorecard assumes k8s cluster configuration.
Also added a scorecard test.

### CODE REVIEW

File: certification/internal/shell/scorecard_basic_check_spec.go [hunk 1]
```diff
@@ -38,6 +38,15 @@ func (p *ScorecardBasicSpecCheck) Validate(bundleImage string) (bool, error) {
 	stdouterr, err := exec.Command("operator-sdk", "scorecard",
 		"--selector=test=basic-check-spec-test",
 		"--output", "json", bundleImage).CombinedOutput()
+	
+	// Scorecard expects K8s Cluster
+	lines := strings.Split(string(stdouterr), "Error:")
+
+	if strings.Contains(lines[len(lines)-1], "error running tests error creating ConfigMap configmaps is forbidden") {
+		log.Error("error running tests error creating ConfigMap configmaps is forbidden")
+        log.Debug("Make sure K8s cluster is configured.")
```
Comment (samira-barouti) [inline certification/internal/shell/scorecard_basic_check_spec.go:47]: indentation is a bit off.

File: certification/internal/shell/scorecard_basic_check_spec.go [hunk 2]
```diff
@@ -38,6 +38,15 @@ func (p *ScorecardBasicSpecCheck) Validate(bundleImage string) (bool, error) {
 	stdouterr, err := exec.Command("operator-sdk", "scorecard",
 		"--selector=test=basic-check-spec-test",
 		"--output", "json", bundleImage).CombinedOutput()
+	
+	// Scorecard expects K8s Cluster
+	lines := strings.Split(string(stdouterr), "Error:")
+
+	if strings.Contains(lines[len(lines)-1], "error running tests error creating ConfigMap configmaps is forbidden") {
+		log.Error("error running tests error creating ConfigMap configmaps is forbidden")
```
Comment (samira-barouti) [inline certification/internal/shell/scorecard_basic_check_spec.go:46]: It would be great to have a descriptive error message.

**Review by samira-barouti — CHANGES_REQUESTED** [review]

**Comment by bcrochet** [comment]
/hold
Revisit when #81 lands

File: certification/internal/shell/scorecard_basic_check_spec.go (hunk 1 above)
Comment (rocrisp) [reply certification/internal/shell/scorecard_basic_check_spec.go:47]: I'm going to close this PR and add the above check to maybe something like cluster configuration check before we even get into scorecard test.  Premise that all scorecard test will fail if the k8s cluster is not configured.

**Comment by rocrisp** [comment]
I'm going to close this PR and add the above check to maybe something like cluster configuration check before we even get into scorecard test.  Premise that all scorecard test will fail if the k8s cluster is not configured.

**Pull request closed without merging**